source/extensions/filters/http/oauth2/filter.cc

```diff
@@ -1235,6 +1240,12 @@ void OAuth2Filter::addResponseCookies(Http::ResponseHeaderMap& headers,
 
   absl::flat_hash_map<std::string, std::string> request_cookies =
       Http::Utility::parseCookies(*request_headers_);
+
+  std::string cookie_delete_secure = "; secure";
+  if (!Runtime::runtimeFeatureEnabled("envoy.reloadable_features.oauth2_secure_cookie_delete")) {
+    cookie_delete_secure = "";
+  }
```

Comment:
Please add a constexpr value for this flag. You can find lots of similar values at this source file. And then, at there, you can create something like this:

```
absl::string_view cookie_delete_secure
if (Runtime::runtimeFeatureEnabled("envoy.reloadable_features.oauth2_secure_cookie_delete")) {
  cookie_delete_secure = SecureCookie;
}
```